helm/mds/templates/deployment.yaml

```diff
@@ -116,7 +116,7 @@ spec:
             value: http://mds-jurisdiction-service
           - name: POLICY_SERVICE_RPC_HOST
             value: http://mds-policy-service
-          - name: TRANSACTIONS_SERVICE_RPC_HOST
+          - name: TRANSACTION_SERVICE_RPC_HOST
```

Comment:
This is the beginning of some renaming I'd like to do, we don't typically use plurality for our service/api names. So this should really be the `mds-transaction-{api | service}`, not `mds-transactions-{api | service}`.